scripts/eks/appsignals/create-slo.sh

```diff
@@ -22,74 +23,55 @@ check_if_loop_failed_and_exit() {
 }
 
 # Add model to aws cli for new aws cloudwatch commands. It is not required after SDK is released
-aws configure add-model --service-model file://slo/monitoring-2010-08-01.normal.json --service-name cloudwatch
+aws configure add-model --service-model file://slo/application-signals-2024-04-15.normal.json --service-name application-signals
 check_if_step_failed_and_exit "There was an error adding the model to aws cli, exiting"
 
 # SLR could be created via console or API. So EnableTopologyDiscovery API is called to enroll topology discovery.
-aws cloudwatch enable-topology-discovery --region $REGION
+aws application-signals start-discovery --region $REGION --endpoint $ENDPOINT
 check_if_step_failed_and_exit "There was an error enabling topology discovery, exiting"
 
 # Pause for synthetics canaries to generate traffic
 echo "Wait 10 minutes for canaries to generate traffic"
-sleep 600
+#sleep 600
```

Comment:
Why comment this line out? We don't need to wait now?

Reply:
We do need to wait when we first create the service. I was testing this with an existing service so commented this line out to speed up the testing. Sending a new PR with that line included.